### DISCUSSION TITLE
HtmlRspackPlugin咨询

### DISCUSSION THREAD

**Discussion by jinphic** [web-infra-dev/rspack#4633] [Q&A]
我们日常开发会依赖 htmlWebpackPluginAfterHtmlProcessing 做一些事情，比如上报、比如处理 inline source ，但是我翻了rspack的文档，没有找到相关的钩子。
1. HtmlRspackPlugin 是不是没有这类钩子 ？
2. 从类型声明来看 HtmlRspackPlugin 是不支持多入口 ？

**Comment by hardfist** [comment]
建议使用html-webpack-plugin, html-rspack-plugin只是临时方案
